Podverse/PVAuth.swift

```diff
@@ -170,26 +173,36 @@ class PVAuth: NSObject {
         }
     }
     
+    fileprivate func notifyLoggingIn() {
+        DispatchQueue.main.async {
```

Comment:
If you are dispatching the notification on the main queue here, is it necessary to dispatch the observers to the main queue as well? I noticed a lot of main dispatches and I think if you do it here, thats good enough